wiki/howToUseSemantics.md

```diff
@@ -0,0 +1,42 @@
+## How to use semantics
+
+To import the project user need to add to pom.xml
+
+    <repositories>
+       <repository>
+           <id>jitpack.io</id>
+           <url>https://jitpack.io</url>
+       </repository>
+    </repositories>
+    <dependency>
+        <groupId>com.github.eiffel-community</groupId>
+        <artifactId>eiffel-remrem-semantics</artifactId>
+        <version>x.x.x</version>
+    </dependency>
+
+Or import the project to pom.xml
+
+    repositories {
+         ...
+         maven { url 'https://jitpack.io' }
+    }
+    dependencies {
+        compile("com.github.eiffel-community:eiffel-remrem-semantics:x.x.x")
+    }
+
+Eiffel REMReM Semantics contains implementation of Eiffel REMReM Protocol Interface. All methods of interface MsgService are implemented by SemanticsService class: generateMsg, getEventId, getEventType, getSupportedEventTypes, getEventTemplate, getServiceName, validateMsg, generateRoutingKey. Also in this class could be implemented more methods depending on the needs of developer.
```

Comment:
this part of the text can be moved to Introduction?

Reply:
Yes but in this content, some of the method names of a class are mentioned. So this can confuse the reader if we move this to the content to the introduction.